Change sonatype maven repository

Since February 2021, new projects should be deployed on `s01.sonatype.org`. See [here](https://central.sonatype.org/publish/publish-guide/#releasing-to-central)

> @rultor release, tag is `0.3.0`

@baudoliver7 Oops, I failed. You can see the full log [here](https://www.rultor.com/t/25244-962447254) (spent 4min)

```
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.sonatype.nexus.client.core.exception.NexusClientErrorResponseException: Unable to find matching profile
    at org.sonatype.nexus.client.rest.jersey.JerseyNexusClient.convertIf400WithErrorMessage (JerseyNexusClient.java:256)
    at org.sonatype.nexus.client.rest.jersey.JerseyNexusClient.convertIfKnown (JerseyNexusClient.java:291)
    at com.sonatype.nexus.staging.client.internal.ExceptionConverter.handleUniformInterfaceException (ExceptionConverter.java:51)
    at com.sonatype.nexus.staging.client.internal.ExceptionConverter.runAndReturn (ExceptionConverter.java:41)
    at com.sonatype.nexus.staging.client.internal.StagingWorkflowV2ServiceImpl.matchProfile (StagingWorkflowV2ServiceImpl.java:117)
    at org.sonatype.nexus.maven.staging.deploy.strategy.AbstractStagingDeployStrategy.selectStagingProfile (AbstractStagingDeployStrategy.java:78)
    at org.sonatype.nexus.maven.staging.deploy.strategy.StagingDeployStrategy.deployPerModule (StagingDeployStrategy.java:72)
    at org.sonatype.nexus.maven.staging.deploy.DeployMojo.execute (DeployMojo.java:192)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[ERROR] 
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
container 1362f10278540564393904ff3c9d8801af6b6290c1e5544479c72328f85fd1ab is dead
Sat Nov  6 13:56:22 CET 2021
```

--- a/pom.xml
+++ b/pom.xml
@@ -69,16 +69,6 @@ SOFTWARE.
     <system>rultor</system>
     <url>https://www.rultor.com/s/lightweight-db</url>
   </ciManagement>
-  <distributionManagement>
-    <snapshotRepository>
-      <id>ossrh</id>
-      <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
-    </snapshotRepository>
-    <repository>
-      <id>ossrh</id>
-      <url>https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-    </repository>
-  </distributionManagement>
   <dependencyManagement>
     <dependencies>
       <dependency>
@@ -115,7 +105,6 @@ SOFTWARE.
         <artifactId>qulice-maven-plugin</artifactId>
         <version>0.19.4</version>
         <configuration>
-          <license>file:${basedir}/LICENSE.txt</license>
           <excludes>
             <exclude>xml:.*</exclude>
           </excludes>
@@ -156,29 +145,15 @@ SOFTWARE.
   </build>
   <profiles>
     <profile>
-      <id>qulice</id>
+      <id>sonatype</id>
       <build>
         <plugins>
           <plugin>
-            <groupId>com.qulice</groupId>
-            <artifactId>qulice-maven-plugin</artifactId>
-            <version>0.19.4</version>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
             <configuration>
-              <license>file:${basedir}/LICENSE.txt</license>
-              <excludes>
-                <exclude>checkstyle:/src/main/resources/.*</exclude>
-                <exclude>findbugs:.*</exclude>
-                <exclude>duplicatefinder:.*</exclude>
-                <exclude>dependencies:.*</exclude>
-              </excludes>
+              <nexusUrl>https://s01.oss.sonatype.org/</nexusUrl>
             </configuration>
-            <executions>
-              <execution>
-                <goals>
-                  <goal>check</goal>
-                </goals>
-              </execution>
-            </executions>
           </plugin>
         </plugins>
       </build>

